scripts/test-utils/harmlessUploadDemos.js

```diff
@@ -0,0 +1,53 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const assert = require('assert/strict');
+const { exec } = require('child_process');
+const { checkInit, uploadProgram } = require('./util.js');
+
+async function main(demoPaths) {
+    // Create connection
+    const provider = new WsProvider('ws://127.0.0.1:9944');
+    const api = await ApiPromise.create({ provider });
+
+    // Create alice account
+    const account = new Keyring({ type: 'sr25519' }).addFromUri('//Alice');
+    // Check that it is root
+    assert.ok((await api.query.sudo.key()).eq(account.addressRaw));
+
+    const isInitialized = checkInit(api);
+
+    for (const pathToDemo of demoPaths) {
+        console.log(`Uploading demo: ${pathToDemo}`);
+
+        const [_programId, messageId] = await uploadProgram(api, account, pathToDemo);
+        await isInitialized(messageId);
```

Comment:
`wrong load ` cannot be initialised. We just check that node does not panic after we try to upload this kind of test.

Reply:
Done